### DISCUSSION TITLE
Can AsyncCommand provide ObservesProperty? (Similar to Prism's DelegateCommand)

### DISCUSSION THREAD

**Discussion by LeoJHarris** [CommunityToolkit/Maui#1464] [New Feature Discussions]
Hi there, 

We have apps using `Prism` which has the `DelegateCommand`. `DelegateCommand` has the `ObservesProperty` which automatically calls the `RaiseCanExecuteChanged` which as I understand can carry a bit more overhead then manually calling that method but removes the need for repetitive code, is there any opportunity that AsyncCommand can implement the same functionality?

https://prismlibrary.com/docs/commands/commanding.html#observesproperty

**Comment by pictos** [comment]
@Sergio0694 I believe this question is for the MVVM toolkit, can you confirm that?

**Comment by CliffAgius** [comment]
Closed and answered
